CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ All notable changes to this project will be documented in this file.
 
 ### :boom: Breaking Change
 
+* feat: metric aggregation temporality controls [#2902](https://github.com/open-telemetry/opentelemetry-js/pull/2902) @seemk
```

Comment:
From an end-user perspective this changelog isn't that helpful. Can you give a description that someone not familiar with the code would be able to more easily understand?

Reply:
Hopefully a little bit more understandable now :slightly_smiling_face: